Makefile

```diff
@@ -88,11 +88,11 @@ ifndef ignore-not-found
 endif
 
 .PHONY: install
-install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+install: verify-manifests manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
```

Comment:
perhaps not needed, given the action will run - but it may be better to warn about this in multiple targets?

Reply:
Yeah, this would fail locally until you committed the changes in which would be annoying